examples/webpack/example-builder.js

```diff
@@ -2,11 +2,39 @@ import assert from 'assert';
 import fs from 'fs';
 import path, {dirname} from 'path';
 import {fileURLToPath} from 'url';
+import eslint from 'eslint';
+import * as espree from 'espree';
```

Comment:
Although it does not make much difference when running in Node, it might me nicer to
```suggestion
import {parse} from 'espree';
```